stdlib/2and3/xml/__init__.pyi

```diff
@@ -1,7 +1,3 @@
-# Stubs for xml.sax (Python 3)
-#
-# NOTE: This dynamically typed stub was automatically generated by stubgen.
-
 class SAXException(Exception):
```

Comment:
This file should actually be `xml/sax/__init__.pyi` (these names don't exist in xml). Let's fix that in a separate PR.